parser.go

```diff
@@ -1172,14 +1172,14 @@ func (p *parser) handleCompoundType(pkgPath, pkgName, typeName string) (*SchemaO
 	}
 
 	sob := &SchemaObject{}
-	switch op {
-	case "not":
+	switch {
+	case strings.EqualFold(op, "not"):
```

Comment:
the convention in the repo so far has been to force everything lower case: https://github.com/launchdarkly/goas/blob/master/parser.go#L328

To me overall it isn't a big difference either way though.

Reply:
thanks for bringing that up, I prefer consistency so I'll update this